src/main/resources/db/migration/V5__Add_stage_to_opportunities.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE opportunities
+ADD stage varchar(400);
```

Comment:
Might be useful to look into storing `stage` as an enum. Even though you have validation this is an extra place that ensures that stages going into the database are one of only a certain number of options.